### DISCUSSION TITLE
Can I generate fluent mapping from database?

### DISCUSSION THREAD

**Discussion by twilightrus** [linq2db/linq2db#4008] [Q&A]
I have migrated database and want to have tool for generate fluent mappings from database. I only need fluent mapping, not a model with attributes.

**Comment by MaceWindu** [comment]
Not yet, but we have it in plans #945

**Comment by alexeytroshkin** [comment]
@MaceWindu Hello, could you please tell me approximately when support for fluent-based generation will be available? Сould not find this information in #945

**Reply by MaceWindu** [reply]
No promises but will try to find time before next release

**Reply by alexeytroshkin** [reply]
Thank you very much for your response

**Comment by MaceWindu** [comment]
Released in 5.1.0
